docs/CONTRIBUTING.md

```diff
@@ -76,8 +76,7 @@ as well.
 In order to run the frontend, you will need to have installed:
 
 - [node](https://nodejs.org/en/) >= 6.10.0
-- [npm](https://www.npmjs.com/get-npm) >= 3.10.10
-- [yarn](https://yarnpkg.com/en/docs/install) >= 0.23.2
+- [npm](https://www.npmjs.com/get-npm) >= 4.0.0
```

Comment:
Just my two cents: running "npm shrinkwrap" with npm5 will simply rename the `package-lock.json` to `npm-shrinkwrap.json`, and will use it for future operations. This lockfile is compatible with npm4! https://docs.npmjs.com/cli/shrinkwrap